debian/patches/use_system_libyaml

```diff
@@ -0,0 +1,58 @@
+Description: Use system libyaml
+Author: Faidon Liambotis <paravoid@debian.org>
+Forwarded: https://github.com/twitter/twemproxy/pull/121
+Last-Update: 2014-06-23
+
+--- a/configure.ac
++++ b/configure.ac
+@@ -196,15 +196,8 @@ AS_IF([test "x$disable_stats" = xyes],
+   [AC_DEFINE([HAVE_STATS], [1], [Define to 1 if stats is not disabled])])
+ AC_MSG_RESULT($disable_stats)
+ 
+-# Untar the yaml-0.1.4 in contrib/ before config.status is rerun
+-AC_CONFIG_COMMANDS_PRE([tar xvfz contrib/yaml-0.1.4.tar.gz -C contrib])
```

Comment:
Unrelatedly, these patches are outdated - twemproxy 0.5.0 is using yaml-0.2.5

Though this is irrelevant since I don't plan on including the changes made for debian packagers in this repo itself for the previously stated reasons.

Reply:
sure. will change that a.s.a.p.
thanks.

Reply:
[new commit 2bccd1a8fcd3f7d419865b0c2797e6ce7370fc0d](https://github.com/steamboatid/twemproxy/commit/2bccd1a8fcd3f7d419865b0c2797e6ce7370fc0d)